shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/segment/dml/TableFactorSegment.java

```diff
@@ -37,6 +38,8 @@
     
     private TableSegment table;
     
+    private SubqueryTableSegment subqueryTableSegment;
```

Comment:
Why do you need `SubqueryTableSegment `? Is the `TableSegment ` not enough?

Reply:
SubqueryTableSegment has been deleted. @tristaZero